src/axom/sina/examples/sina_tutorial.cpp

```diff
@@ -130,13 +130,15 @@ void gatherAllData(axom::sina::Record &record)
 void save(axom::sina::Document const &doc)
 {
   axom::sina::saveDocument(doc, "my_output.json");
+  axom::sina::saveDocument(doc, "my_output.hdf5", axom::sina::Protocol::HDF5);
 }
 //! [end io write]
 
 //! [begin io read]
 void load()
 {
-  axom::sina::Document doc = axom::sina::loadDocument("my_output.json");
+  axom::sina::Document doc1 = axom::sina::loadDocument("my_output.json");
+  axom::sina::Document doc2 = axom::sina::loadDocument("my_output.json", axom::sina::Protocol::HDF5);
```

Comment:
I think this should this be "my_output.hdf5" as the first argument to `loadDocument`